index.js

```diff
@@ -45,18 +45,18 @@ const keyFilePath = path.resolve(commander.configFile || '.gcloud.json'),
     gcloudConfig.bucket,
     remotePath
   ),
-  slack = new Slack(gcloudConfig.slackWebHook);
+  slack = new Slack(gcloudConfig.slackWebHook),
+  metadata = gcloudConfig.metadata || {
+    cacheControl: 'no-cache'
+  };
 
 let storage, bucket, files,
   asyncTasks = [];
 
 storage = gcloud({
   projectId: gcloudConfig.projectId,
   bucket: gcloudConfig.bucket,
-  keyFilename: keyFilePath,
-  metadata: {
```

Comment:
Removed the metadata for the storage object as I didn't find reference to it in the docs. https://googlecloudplatform.github.io/gcloud-node/#/docs/v0.36.0/gcloud